src/renderer/bootstrap.tsx

```diff
@@ -80,9 +81,14 @@ export async function bootstrap(App: AppComponent) {
       window.location.href = "about:blank";
     }
   });
+
+  const cacheProps = { nonce: "lens", cacheKey: "lens" };
+
   render(<>
     {isMac && <div id="draggable-top" />}
-    <App />
+    <StyleCache {...cacheProps}>
```

Comment:
Passing props this way allows to ignore `NonceProvider` stale types. Current version of it requires `cacheKey`, but type definitions doesn't show that.

Reply:
I have filled https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52073 to fix this, hopefully it gets merged soon.

Reply:
Thanks! That's cool! 🚀